src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/jpa/entity/AssessmentEntity.kt

```diff
@@ -276,6 +276,7 @@ class ApprovedPremisesAssessmentEntity(
   referralHistoryNotes: MutableList<AssessmentReferralHistoryNoteEntity>,
   schemaUpToDate: Boolean,
   isWithdrawn: Boolean,
+  var createdFromAppeal: Boolean,
```

Comment:
I don't _think_ this needs to be a var?

Reply:
I still can't get the difference straight in my head! All sorted now 👍